desktop/libs/indexer/src/indexer/indexers/sql.py

```diff
@@ -182,6 +184,11 @@ def create_table_from_a_file(self, source, destination, start_time=-1, file_enco
 
     if external_path.lower().startswith("abfs"): #this is to check if its using an ABFS path
       external_path = abfspath(external_path)
+    elif external_path.lower().startswith("ofs") or source_path.lower().startswith("ofs"):  #this is to check if its using an OFS path
+      # as of now ozone has 3 managers and impala/hive don't know on which manager
+      # they need to ping hence we are sending the ozone_service_id with path
+      external_path = external_path[:6] + ozone_service_id + external_path[5:]
+      source_path = source_path[:6] + ozone_service_id + source_path[5:]
```

Comment:
My understanding is that you are trying to insert the namenode (or ozone manager) url in between to make the path explicit. What happens if the `external_path` is already a full url? Or is that something that can never happen.

Reply:
Yes, so external_path is always like -> ofs://volume/bucket/key (because we are taking this path form file chooser and it will always return the path like mentioned above).
For hive and impala, we need to insert the ozone manager in the path explicitly.